Ollamac/ViewModels/MessageViewModel.swift

```diff
@@ -98,6 +98,10 @@ final class MessageViewModel {
         try? self.modelContext.saveChanges()
     }
     
+    func resetHistory() {
+        self.messages = []
```

Comment:
Are you trying to clear the messages from the persistent storage? Just to clarify, the current action only removes them from the array, not from the persistent storage.

Reply:
Oh I see, my goal was to add a reset button and completely clear that chat from both UI and storage.

Reply:
Would you mind creating a separate PR for this functionality? Since it seems like a significant feature, having it in its own PR would make it easier for us to review and test thoroughly. It's always good to keep big changes isolated so we can manage them better.